test-app.gradle

```
@@ -33,11 +33,23 @@ private static void applySettings(Settings settings) {
         .projectDir = new File("${projectDir}/android/support")
 }
 
+// TODO: Remove when `@react-native-community/cli` 6.0+ is required. See also
+// https://github.com/react-native-community/cli/commit/fa0d09b2c9be144bbdff526bb14f171d7ddca88e.
```

```suggestion
// https://github.com/react-native-community/cli/commit/fa0d09b2c9be144bbdff526bb14f171d7ddca88e
```
removing the dot at the end - with it, it's a broken link